src/main/java/edu/hawaii/its/groupings/controller/HomeController.java

```diff
@@ -230,4 +231,9 @@ public String dynamicModal() {
     public String groupingOwnersModal() {
         return "modal/groupingOwnersModal";
     }
+
+    @GetMapping(value = "/modal/{modalName}")
+    public String openModalPath(@PathVariable String modalName) {
+        return "modal/" + modalName;
+    }
```

Comment:
Thanks for this addition. With this change I think we can remove all the modal functions above. Thanks.

Reply:
Also make sure to update the tests for HomeController.

Reply:
Will this be another ticket? Removal of modal functions above this one that will replace, and update HomeController?

Reply:
Yep, make another ticket for that.